crates/bitwarden-vault/src/cipher/cipher.rs

```diff
@@ -453,7 +453,7 @@ impl CipherView {
     pub fn decrypt_fido2_credentials(
         &self,
         enc: &dyn KeyContainer,
-    ) -> Result<Vec<Fido2CredentialView>, CipherError> {
+    ) -> Result<Vec<Fido2CredentialView>, CryptoError> {
```

Comment:
@coroiu Do you remember why we used `CipherError` here instead of the lower level `CryptoError`?

Reply:
Hmm, no idea, I think this was mostly you and Dani that did this